docs/distributed-tracing-guide.asciidoc

```diff
@@ -23,22 +26,23 @@ var apm = initApm({
 })
 ----
 
-
 This effectively tells the agent to add the distributed tracing HTTP header (`elastic-apm-traceparent`) 
 to requests made to `https://api.example.com`.
 
+To read more about cross-origin requests and why this process is necessary,
```

Comment:
Can we also mentioned here that distributed tracing headers would be only appended to `API` calls? so it becomes clear and we can link to the below dynamically generated html heading incase users wants to view the full trace?

Reply:
Can you check the latest commit and let me know if that covers it?

Reply:
Yes looks good Brandon.